runit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Does not work directly on windows. It is intended to run inside the docker container which is an Ubuntu image. Once built, the docker container does run on windows.